packages/terra-paginator/src/Paginator.module.scss

```diff
@@ -51,8 +51,9 @@
     }
 
     .is-selected {
+      color: #000;
```

Comment:
Let's set this to `color: var(--terra-paginator-nav-link-selected-color, inherit);`. This will inherit the parent elements color which is likely inherited from [base](https://github.com/cerner/terra-core/blob/master/packages/terra-base/src/Base.scss#L14). Then we should be good to just have 1 instance of color in this selector.